src/transformers/modeling_transfo_xl.py

```diff
@@ -221,20 +219,20 @@ def forward(self, inp):
 
 class RelPartialLearnableMultiHeadAttn(nn.Module):
     def __init__(
-        self,
-        n_head,
-        d_model,
-        d_head,
-        dropout,
-        dropatt=0,
-        tgt_len=None,
-        ext_len=None,
-        mem_len=None,
-        pre_lnorm=False,
-        r_r_bias=None,
-        r_w_bias=None,
-        output_attentions=False,
-        layer_norm_epsilon=1e-5,
+            self,
+            n_head,
+            d_model,
+            d_head,
+            dropout,
+            dropatt=0,
+            tgt_len=None,
+            ext_len=None,
+            mem_len=None,
+            pre_lnorm=False,
+            r_r_bias=None,
+            r_w_bias=None,
+            output_attentions=False,
+            layer_norm_epsilon=1e-5,
```

Comment:
you have a lot of whitespace only changes

Reply:
Yeah it's a bit annoying, probably comes from running the auto-format function of Pycharm, will remove them